experimental/globaldata/src/data/core_helloworld_v1/en.data.rs

```diff
@@ -0,0 +1,10 @@
+#[macro_export]
```

Comment:
Wrapping each data expression in a macro is a lot of cognitive overhead, and probably also compile time overhead.

Reply:
The cognitive overhead is being moved from the call site (people need to learn how data providers work) to an internal crate (that only us and power users will see).